volttron/platform/web/admin_endpoints.py

```diff
@@ -258,7 +254,7 @@ def __deny_csr_api(self, common_name):
 
     def __delete_csr_api(self, common_name):
         try:
-            self._certs.delete_csr(common_name)
+            self._rpc_caller.call(AUTH, 'delete_authorization_failure', common_name)
```

Comment:
.get(timeout=2) missing

Reply:
Get is used for return values. "delete_authorization_failure" does not have a return value. I could add a wait(timeout=2)